backend/config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
+  namespace :api do
+    namespace :v1 do
+      get 'invitation-info', to: 'invitation_info#index'
+      get 'guest-info', to: 'guest_info#index'
+      post 'guest-info', to: 'guest_info#index'
```

Comment:
```suggestion
      post 'guest-info', to: 'guest_info#create'
```
とかかな
ただ、updateの場合もあるので、upsertの方がいいかも